builder/bundle_control.go

```diff
@@ -831,53 +752,35 @@ func createBundleFile(bundle string, path string) error {
 }
 
 // EditBundles copies a list of bundles from upstream-bundles to local-bundles
-// (if they are not already there) or creates a blank template if they are new,
-// and launches an editor to edit them. Passing true for 'suppressEditor' will
-// suppress the launching of the editor (and just do the copy or create, if
-// needed), and 'add' will also add the bundles to the mix.
-func (b *Builder) EditBundles(bundles []string, suppressEditor bool, add bool, git bool) error {
+// (if they are not already there) or creates a blank template if they are new.
+// 'add' will also add the bundles to the mix.
+func (b *Builder) EditBundles(bundles []string, add bool, git bool) error {
 	// Fetch upstream bundle files if needed
 	if err := b.getUpstreamBundles(b.UpstreamVer, true); err != nil {
 		return err
 	}
 
-	editorCmd, err := getEditorCmd()
-	if err != nil {
-		fmt.Println("Cannot find a valid editor (see usage for configuration). Copying to local-bundles only.")
-		suppressEditor = true
-	}
-
 	for _, bundle := range bundles {
 		path, _ := b.getBundlePath(bundle)
 		if !b.isLocalBundle(path) {
 			localPath := filepath.Join(b.Config.Mixer.LocalBundleDir, bundle)
 
 			if path == "" {
-				// Bunlde not found upstream, so create new
-				if err = createBundleFile(bundle, localPath); err != nil {
+				// Bundle not found upstream, so create new
+				if err := createBundleFile(bundle, localPath); err != nil {
```

Comment:
You can just declare `var err error` on line 762 and not have to edit any of these lines below.